app/src/main/java/org/sopt/and/presentation/signin/SignInScreen.kt

```diff
@@ -84,11 +102,11 @@ fun SignInScreen(
 
             Spacer(modifier = Modifier.height(30.dp))
 
-            EmailInputField(
-                value = uiState.email,
-                onValueChange = { signInViewModel.onEmailChange(it) },
-                placeholder = "이메일 주소 또는 아이디",
-                isError = uiState.errorMessage?.contains("이메일") == true
+            TextInputField(
+                value = uiState.username,
+                onValueChange = { signInViewModel.onUsernameChange(it) },
```

Comment:
```suggestion
                onValueChange = signInViewModel::onUsernameChange,
```
이런 방법으로도 작성할 수 있어요. 컴포즈에서는 이러한 방법이 확실한 장점이 있는 반면, 단점도 있어요. 한번 찾아보시고 상황에 맞게 사용하시면 좋을 것 같네요